ui/app/models/pki/tidy.js

```diff
@@ -49,6 +49,24 @@ export default class PkiTidyModel extends Model {
   })
   intervalDuration; // auto-tidy only
 
+  @attr('string', {
+    editType: 'ttl',
+    helperTextEnabled:
+      'Sets the min_startup_backoff_duration field which forces the minimum delay after Vault startup auto-tidy can run',
+    hideToggle: true,
+    formatTtl: true,
+  })
+  minStartupBackoffDuration;
```

Comment:
Just noticed we have comments on the other auto only params, so doesn't hurt to add here:
```suggestion
  minStartupBackoffDuration; // auto-tidy only 
```